src/libraries/System.Private.CoreLib/src/System/SearchValues/ProbabilisticCharSearchValues.cs

```diff
@@ -14,16 +13,6 @@ internal sealed class ProbabilisticCharSearchValues : SearchValues<char>
 
         public ProbabilisticCharSearchValues(scoped ReadOnlySpan<char> values)
         {
-            if (Vector128.IsHardwareAccelerated && values.Length < 8)
-            {
-                // ProbabilisticMap does a Span.Contains check to confirm potential matches.
-                // If we have fewer than 8 values, pad them with existing ones to make the verification faster.
-                Span<char> newValues = stackalloc char[8];
-                newValues.Fill(values[0]);
-                values.CopyTo(newValues);
-                values = newValues;
-            }
-
```

Comment:
Why is this no longer relevant?

Reply:
It is. I moved it up to `SearchValues.Create` as it applies to both Probabilistic implementations.

Reply:
Nevermind, I see you moved it.